_source/logzio_collections/_log-sources/platform-service/zipkin.md

```diff
@@ -46,6 +46,13 @@ For most users, these won't be an issue, but they're still good to know:
 
     Download the [Zipkin-Logz.io Trace Storage](https://github.com/logzio/zipkin-logzio/releases) jar to the same directory.
 
+    ```shell
+    RELEASE_JAR=$(curl -s https://api.github.com/repos/logzio/zipkin-logzio/releases/latest \
+      | grep "tag_name" \
+      | awk '{print "https://github.com/logzio/zipkin-logzio/archive/" substr($2, 2, length($2)-3) ".jar"}') \
+      ; curl -L -o zipkin-logzio.jar $RELEASE_JAR
```

Comment:
It doesn't work for me... I see the correct address in RELAESE_JAR but it doesn't download the file, perhaps use wget instead?

Reply:
Please re-review. Should be good.